plasma/root_chain/contracts/RLPEncode.sol

```diff
@@ -0,0 +1,127 @@
+pragma solidity ^0.4.23;
+pragma experimental ABIEncoderV2;
```

Comment:
Is this used in the contract?

Reply:
yes, without it, `bytes[]` in `encodeList` is not support